src/quo/foundations/colors.cljs

```diff
@@ -245,8 +245,11 @@
 ;;;; Networks
 (def ^:private networks
   {:ethereum "#758EEB"
+   :eth      "#758EEB"
```

Comment:
I think that's fine, two maps is a bit neater, but definitely I would extract the colour if you want to go this way i.e:

```
(def ... networks
   {:ethereum ethereum-colour
    :eth ethereum-color
   ....
``` 
Though i have a slight preference to two maps and translate the key, but no strong opinion